packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -56,7 +56,17 @@ export class LicenseManager {
 			throw new Error('Invalid license key')
 		}
 		const licenseInfo = JSON.parse(util.encodeUTF8(decoded))
-		return licenseInfoValidator.validate(licenseInfo)
+		try {
+			return licenseInfoValidator.validate(licenseInfo)
+		} catch (e: any) {
+			if (e.message.includes('Unexpected property')) {
```

Comment:
It wouldn't give you the extra message (not sure how important that is) but in case u missed it you can do `T.object({ ... }).allowUnknownProperties()`

Reply:
Yeah, I kinda do that a [bit further down](https://github.com/tldraw/tldraw/pull/4053/files#diff-6e7aadaa37ccd9daf67f2589ea799f33bf1c67b5a47ace64d4096ee40f460358R69). This is just a hacky way to detect that we have some unexpected properties, which we can then communicate to the devs - they might not be aware that the licenses key they are using might unlock certain feature for them.

Reply:
There might be a better way to do that though? 😄

Reply:
Move to using an array for representing the different properties so this part is obsolete now.